src/main/kotlin/racingcar/utils/RandomNumber.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar.utils
+
+import kotlin.random.Random
+
+object RandomNumber {
+    private const val MIN_RANDOM_NUMBER = 0
+    private const val MAX_RANDOM_NUMBER = 9
+
+    fun generator() = Random.nextInt(MIN_RANDOM_NUMBER, MAX_RANDOM_NUMBER)
```

Comment:
```suggestion
    fun generate() = Random.nextInt(MIN_RANDOM_NUMBER, MAX_RANDOM_NUMBER)
```

이건 팀마다 컨벤션이 다를수도 있는데,
저는 메서드의 경우에는 명사가 아니라 동사로 하는 것을 선호하는 편입니다.